package.json

```diff
@@ -40,6 +40,7 @@
     "font-awesome": "4.4.0",
     "htmlparser2": "3.8.3",
     "jquery": "2.1.4",
+    "lodash": "4.5.1",
     "markdown-it": "4.4.0",
```

Comment:
If we're bringing in lodash (which I'm in favor of), shouldn't we remove underscore?